tools/cli/src/cli/common.rs

```diff
@@ -91,6 +91,18 @@ impl CommonOpt {
             false => client_builder,
         };
 
+        let client_builder = match self.accept_invalid_certs {
+            true => {
+                warn!(
+                    "Accepting invalid certs, this can lead to a password leak"
```

Comment:
Maybe just change this to "TLS Certificate Verification disabled!!! This can lead to credential and account compromise!!!"